src/main/java/com/iheartradio/android/modules/crowdcontrol/AtomParameter.java

```diff
@@ -1,4 +1,4 @@
-package com.lotame.android;
```

Comment:
i think we should keep package structure the same as well - i think it's helpful to know, even in imports, that this originates from lotame